packages/release-tool/src/index.ts

```diff
@@ -18,95 +16,43 @@ const {
   lte: semverLte,
 } = semver;
 const exec = promisify(child_process.exec);
-const execFile = promisify(child_process.execFile);
-
-const options = commandLineArgs([
-  {
-    name: "type",
-    defaultOption: true,
-  },
-  {
-    name: "preid",
-  },
-  {
-    name: "check-commits",
-    type: Boolean,
-  },
-]);
-
-const validReleaseValues = [
-  "major",
-  "minor",
-  "patch",
-];
-const validPrereleaseValues = [
-  "premajor",
-  "preminor",
-  "prepatch",
-  "prerelease",
-];
-const validPreidValues = [
-  "alpha",
-  "beta",
-];
+const spawn = promisify(child_process.spawn);
 
-const errorMessages = {
-  noReleaseType: `No release type provided. Valid options are: ${[...validReleaseValues, ...validPrereleaseValues].join(", ")}`,
-  invalidRelease: (invalid: string) => `Invalid release type was provided (value was "${invalid}"). Valid options are: ${[...validReleaseValues, ...validPrereleaseValues].join(", ")}`,
-  noPreid: `No preid was provided. Use '--preid' to specify. Valid options are: ${validPreidValues.join(", ")}`,
-  invalidPreid: (invalid: string) => `Invalid preid was provided (value was "${invalid}"). Valid options are: ${validPreidValues.join(", ")}`,
-  wrongCwd: "It looks like you are running this script from the 'scripts' directory. This script assumes it is run from the root of the git repo",
-};
+const repoRoot = (await exec("git rev-parse --show-toplevel")).stdout.trim();
 
-if (!options.type) {
-  console.error(errorMessages.noReleaseType);
+if (process.cwd() !== repoRoot) {
+  console.error("It looks like you are running this script from the 'scripts' directory. This script assumes it is run from the root of the git repo");
   process.exit(1);
 }
 
-if (validReleaseValues.includes(options.type)) {
-  // do nothing, is valid
-} else if (validPrereleaseValues.includes(options.type)) {
-  if (!options.preid) {
-    console.error(errorMessages.noPreid);
-    process.exit(1);
-  }
+const currentVersion = new SemVer(JSON.parse((await readFile("./lerna.json", "utf-8"))).version);
 
-  if (!validPreidValues.includes(options.preid)) {
-    console.error(errorMessages.invalidPreid(options.preid));
-    process.exit(1);
-  }
-} else {
-  console.error(errorMessages.invalidRelease(options.type));
-  process.exit(1);
-}
-
-if (basename(process.cwd()) === "scripts") {
-  console.error(errorMessages.wrongCwd);
-}
+await spawn("npm", ["run", "bump-version"], {
```

Comment:
do you want the `--force-publish` option here?

Reply:
Thinking about this.

I think it would be better if we tried to find a way around this. Maybe by cherry-picking the commits before hand when on not-master. Will try this.